### ISSUE TITLE
DoesNotExist error raised when instantiating composite key model with no parameters

### ISSUE THREAD

**Issue by khoobks-gaiaresources** [simone/django-compositekey#16]
This issue is a very minor issue that can be worked around relatively easily.

The issue is caused when creating a new instance of a model using composite keys without any parameters.

This is best demonstrated with an example

``` python
# models.py
from django.db import models
from compositekey import db


class Author(models.Model):
    name = models.CharField(max_length=100)


class Book(models.Model):
    title = models.CharField(max_length=100)


class BookAuthor(models.Model):
    id = db.MultiFieldPK('author', 'book')
    author = models.ForeignKey('Author')
    book = models.ForeignKey('Book')
```

``` python
# Using the django shell
>>> author = Author()
>>> author.name = "Charles Darwin"
>>> author.save()
>>> 
>>> book = Book()
>>> book.title = "Origin of the Species"
>>> book.save()
>>> 
>>> ba = BookAuthor()
Traceback (most recent call last):
  File "<console>", line 1, in <module>
  File "/ssd/workspace/OpenSource/django-compositekey-fixed/VirtualEnv/local/lib/python2.7/site-packages/compositekey/db/models/base.py", line 108, in patched_model_init
    self.pk
  File "/ssd/workspace/OpenSource/django-compositekey-fixed/VirtualEnv/local/lib/python2.7/site-packages/django/db/models/base.py", line 428, in _get_pk_val
    return getattr(self, meta.pk.attname)
  File "/ssd/workspace/OpenSource/django-compositekey-fixed/VirtualEnv/local/lib/python2.7/site-packages/compositekey/db/models/fields/wrap.py", line 25, in _get
    setattr(obj, cache_name, assemble_pk(*[f.get_prep_value(getattr(obj, f.name)) for f in fields]))
  File "/ssd/workspace/OpenSource/django-compositekey-fixed/VirtualEnv/local/lib/python2.7/site-packages/django/db/models/fields/related.py", line 343, in __get__
    raise self.field.rel.to.DoesNotExist
DoesNotExist
```

The workaround for this issue is to simply create the BookAuthor using keyword arguments

``` python
>>> ba = BookAuthor(author=author, book=book)
>>> ba.save()
>>> 
```

**Comment by simone** [comment]
The workaround is perfect :-D
but we have to add an try: except: in the patched init of the model.

**Comment by simone** [comment]
fixed in 1.4.x branch
